src/components/ProductCard/ProductCard.jsx

```diff
@@ -30,4 +25,9 @@ const ProductCard = () => {
   );
 };
 
+ProductCard.prototype={
```

Comment:
This is an important mistake. "prototype" - is absolutely wrong. For the types you have to write **"propTypes"**